kubetest/main.go

```diff
@@ -352,7 +352,7 @@ func complete(o *options) error {
 	if o.logexporterGCSPath != "" {
 		o.testArgs += fmt.Sprintf(" --logexporter-gcs-path=%s", o.logexporterGCSPath)
 	}
-	if err := prepare(o); err != nil {
+	if err := control.XMLWrap(&suite, "Prepare", func() error { return prepare(o) }); err != nil {
```

Comment:
Can we also wrap getDeployer(o) as well? I can imagine some errors happening there as well.

Reply:
Wrapped getDeployer call as well.